src/Azure.Functions.Cli/Diagnostics/ColoredConsoleLogger.cs

```diff
@@ -0,0 +1,89 @@
+﻿using System;
+using System.Collections.Generic;
+using Colors.Net;
+using Microsoft.Azure.WebJobs.Script;
+using Microsoft.Extensions.Logging;
+using static Azure.Functions.Cli.Common.OutputTheme;
+
+namespace Azure.Functions.Cli.Diagnostics
+{
+    public class ColoredConsoleLogger : ILogger
+    {
+        private readonly Func<string, LogLevel, bool> _filter;
+        private readonly string _category;
+
+        public ColoredConsoleLogger(string category, Func<string, LogLevel, bool> filter)
+        {
+            _category = category;
+            _filter = filter;
+            ColoredConsole.Out = new ColoredConsoleWriter(Console.Out);
+        }
```

Comment:
Is this needed? the other class was doing this to then null out the default Console.Out to prevent anyone else from writing to it.

Reply:
No, it's not needed.